### DISCUSSION TITLE
Quickly cycling through skins may result in having 2 skins active at once

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#15037] [Q&A] [answered]
https://user-images.githubusercontent.com/53081750/136710780-1dbe90c2-a83b-4d56-8718-3bc7d9b7bad8.mp4

version 2021.1006.1

logs:
[network.log](https://github.com/ppy/osu/files/7319905/network.log)
[performance.log](https://github.com/ppy/osu/files/7319906/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7319907/runtime.log)
[database.log](https://github.com/ppy/osu/files/7319911/database.log)

**Comment by peppy** [comment, marked as answer]
How did you create this issue? It should have directed you to make a discussion.

If you intended to make an issue, please attach logs at least.

**Reply by Theighlin** [reply]
I did intend to create an issue because I didn't feel uncertain about what this was, i saw this as a bug, it's not a feature proposal or a question or arguing about lazer stuff. If this is a problem i will open discussions instead from here onwards. I edited the opening post with logs.

**Reply by peppy** [reply]
Thanks. Have PR'd a fix at #15045.

**Comment by smoogipoo** [comment]
Have you checked to make sure the skin doesn't actually have duplicate elements inside it, via the skin editor? Does it fix after re-entering gameplay?

**Reply by Theighlin** [reply]
Yes, i'm 100% sure the skin doesn't have duplicate elements and no, i did no modify it in the skin editor before. It's also got different styles health bar and combo counter, i don't even know if that's something a user can do? It does fix after re entering gameplay.
